### ISSUE TITLE
java.lang.NoClassDefFoundError: Could not initialize class com.jogamp.opengl.GLProfile

### ISSUE THREAD

**Issue by Mishkun** [gephi/gephi#1356]
[messages.txt](https://github.com/gephi/gephi/files/123025/messages.txt)

Got this just after startup. On my second computer I can't reproduce that issue
```
java.lang.NoClassDefFoundError: Could not initialize class com.jogamp.opengl.GLProfile
	at org.gephi.visualization.swing.GLAbstractListener.getCaps(GLAbstractListener.java:174)
	at org.gephi.visualization.swing.NewtGraphCanvas.<init>(NewtGraphCanvas.java:66)
	at org.gephi.visualization.VizController.createNewtCanvas(VizController.java:304)
	at org.gephi.visualization.VizController.initInstances(VizController.java:127)
	at org.gephi.visualization.VizController.getInstance(VizController.java:90)
	at org.gephi.visualization.component.VizBarController.createDefaultGroups(VizBarController.java:99)
	at org.gephi.visualization.component.VizBarController.<init>(VizBarController.java:88)
	at org.gephi.visualization.component.GraphTopComponent.initCollapsePanel(GraphTopComponent.java:118)
	at org.gephi.visualization.component.GraphTopComponent.access$000(GraphTopComponent.java:77)
	at org.gephi.visualization.component.GraphTopComponent$1$1.run(GraphTopComponent.java:99)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:251)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:733)
	at java.awt.EventQueue.access$200(EventQueue.java:103)
	at java.awt.EventQueue$3.run(EventQueue.java:694)
	at java.awt.EventQueue$3.run(EventQueue.java:692)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:703)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:159)
[catch] at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)
```

**Comment by totetmatt** [comment]
[ Windows 7 - Java 1.8 ]
I got similar issue when trying  `package org.gephi:gephi-maven-plugin:run` for plug-in development

Nothing at first but then got an issue from my anti-virus (Avira) that a temp file of (jogamp blbalbal) get blocked because of  `HEUR/APC (Cloud) `.

I've removed the antivirus, relaunched the build and works.

Tried serveral time with and without anti-virus = Same results each time, the antivirus sounds to block the usage of the temporary file created by jogl.
So sounds like
